src/main/java/frc/robot/subsystems/IntakeSystem.java

```diff
@@ -0,0 +1,39 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class IntakeSystem extends SubsystemBase {
+    private CANSparkMax motor;
+    private DigitalInput limitSwitch;
+
+    public IntakeSystem(){
+        motor = new CANSparkMax(RobotMap.INTAKE_MOTOR, CANSparkLowLevel.MotorType.kBrushless);
```

Comment:
Our standard operating procedures require restoring SparkMax to factory default settings after initialization.

Add a call to `motor.restoreFactoryDefaults()`

Reply:
done